src/include/stir/RadionuclideDBProcessor.h

```diff
@@ -0,0 +1,109 @@
+/*!
+
+  \file
+  \ingroup projdata
+  \brief Declaration of class stir::RadionuclideDBProcessor
+    
+  \author Daniel Deidda
+  \author Kris Thielemans
+      
+*/
+/*
+    Copyright (C) 2021, National Physical Laboratory
+    Copyright (C) 2021, University Colleg London
+    See STIR/LICENSE.txt for details
+*/
+
+#ifndef __stir_RADIONUCLIDEDBPROCESSOR_H
```

Comment:
To me, this class should be called `RadioNuclideDataBase` or just `RadionuclideDB`